docs/source/guide/guide-frontends.myst

```diff
@@ -0,0 +1,29 @@
+---
+jupytext:
+  text_representation:
+    extension: .myst
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.11.4
+kernelspec:
+  display_name: Python 3
+  language: python
+  name: python3
+---
+
+# Mitiq Frontends
+
+Mitiq can accept quantum circuit representations in a variety of formats from different tools or platforms.
+These input formats are referred to here as "frontends".
+The main four frameworks currently supported are Cirq, Qiskit, PyQuil, and Braket, with Cirq being used for the internal implementations of the mitigation methods.
```

Comment:
To make it easier to maintain, use

```python
import mitiq
print(mitiq.SUPPORTED_PROGRAM_TYPES)
```

instead of writing them out.

I would also mention `mitiq.QPROGRAM` = intersection of `mitiq.SUPPORTED_PROGRAM_TYPES` and installed programs.

Reply:
+1 on this.